cirq-superstaq/cirq_superstaq/service_test.py

```diff
@@ -64,6 +64,37 @@ def test_counts_to_results() -> None:
     assert result.histogram(key="01") == collections.Counter({0: 50, 3: 50})
 
 
+def test_validate_cirq_circuit() -> None:
+    qubits = [cirq.LineQubit(i) for i in range(2)]
+    circuit = cirq.Circuit(cirq.H(qubits[0]), cirq.CNOT(qubits[0], qubits[1]))
+
+    with pytest.raises(
+        ValueError,
+        match="Invalid 'circuits' input. Must be a `cirq.Circuit` or a "
+        "sequence of `cirq.Circuit` instances.",
+    ):
+        css.service._validate_cirq_circuit("circuit_invalid")
+
+    with pytest.raises(
+        ValueError,
+        match="Invalid 'circuits' input. Must be a `cirq.Circuit` or a "
+        "sequence of `cirq.Circuit` instances.",
+    ):
+        css.service._validate_cirq_circuit([circuit, "circuit_invalid"])
+
+
+def test_validate_get_counts() -> None:
+
+    with pytest.raises(TypeError, match="Repetitions must be an integer."):
```

Comment:
I would also add a test for a string that you'd expect to pass like "10" or float (100.0)

Reply:
Currently, It'll accept "10" and floats like 100.0 since they can be safely cast into integers. But I added some more invalid inputs in 278940a

Reply:
Yeah, I was suggesting to test valid inputs

Reply:
Added your suggestion with commit dab3ed2